pkg/driver/wrapper.go

```diff
@@ -1,14 +1,19 @@
 package driver
 
 import (
+	"github.com/google/uuid"
 	"github.com/pion/mediadevices/pkg/io/audio"
 	"github.com/pion/mediadevices/pkg/io/video"
 	"github.com/pion/mediadevices/pkg/prop"
-	uuid "github.com/satori/go.uuid"
 )
 
 func wrapAdapter(a Adapter, info Info) Driver {
-	id := uuid.NewV4().String()
+	generator, err := uuid.NewRandom()
```

Comment:
`uuid.NewRandom()` returns binary UUID data and it's not generator. It would be better to be renamed.